### ISSUE TITLE
[Rift][Suggestion] One way rift option to not delete

### ISSUE THREAD

**Issue by Dragonias** [zephyrkul/FluffyCogs#52]
This is a simple suggestion as said in the title basicly we'd like to make a bug report channel so that if severe bugs that could be abused get sent in there would get auto deleted by another system but the downside to that is that with the way rift is setup right now it would delete it in the chosen rifted channel too, Instead it would be nicer if it could be set up so that the bot just grabs it and leaves it into the rifted channel if chosen so.

**Comment by zephyrkul** [comment]
Run-on sentences are extremely hard to read.

To clarify, you want a rift option to avoid deleting the bot's relayed message if the original author's message is deleted?

**Comment by Dragonias** [comment]
Indeed, sorry for the wall of text 😅 basicly yeah I want a option for rift that where if the authors message is deleted it stays in the relayed channel, As we would use it for severe bugs for example that not every user would see.
